### ISSUE TITLE
call tile layout broken on narrow screens in safari

### ISSUE THREAD

**Issue by ara4n** [vector-im/element-web#18398]

**Comment by SimonBrandner** [comment]
@gaelledel, how should this behave on narrower screens, perhaps like they look on Android/iOS?

**Comment by gaelledel** [comment]
Yes, We could have them looking like the mobile ones. I'll provide references for you in a moment

**Comment by gaelledel** [comment]
https://www.figma.com/file/V6m2z0oAtUV1l8MdyIrAep/VoIP?node-id=5960%3A140398
